pancancer_classifier.py

```diff
@@ -248,8 +248,8 @@ def get_threshold_metrics(y_true, y_pred, tissue='all'):
 sample_freeze = pd.read_csv(sample_freeze_file)
 
 # Construct data for classifier
-common_genes = set(mutation_df.columns.tolist()).intersection(genes)
-common_genes = list(common_genes.intersection(rnaseq_df.columns.tolist()))
+common_genes = set(mutation_df.columns).intersection(genes)
```

Comment:
`.intersection()` above is a method specific to sets.

i convert the pandas Series to a set in order to define which genes are in common to both lists.

Alternatively I could have used `.isin()` to find the genes in common. I think this would work too:

```python
common_genes = mutation_df[mutation_df.columns.isin(genes)].columns
common_genes = rnaseq[rnaseq.columns.isin(common_genes)].columns
```

This may work too and be more elegant than other solutions? I am not sure.

```python
common_genes = list(set(mutation_df.columns) & set(rnaseq.columns) & set(genes))
```

what do you think?